Scripts/Chatbot/chatbot.js

```diff
@@ -1,32 +1,133 @@
 const chatbotToggle = document.querySelector(".chat-toggle");
+const chatCloseButton = document.querySelector(".close-icon");
 
 let userMessage;
 
-const createChatLi = (message , className) => {
+const responses = {
+    "Hi": "Hello! How can I assist you today?",
+    "Hello":"Hi! How can I assist you today?",
+    "Eita": "Hello! How can I assist you today?",
+    "Huzit": "Ola! How can I assist you today?",
+    "Howzit": "Exe! How can I assist you today?",
+    "Ola": "Sho! How can I assist you today?",
+    "Exe": "Ola! How can I assist you today?",
+    "Fede": "Sho! How can I assist you today?",
+    "How are you": "I am doing great how about you?",
+    "How are you doing": "I am doing great how about you?",
+    "How are you feeling": "I am doing great how about you?",
+    "I am doing great": "Glad to know! How can i be of assistance today?",
+    "I am doing well": "Glad to know! How can i be of assistance today?",
+    "I'm good": "Glad to know! How can i be of assistance today?",
+    "I am okay": "Glad to know! How can i be of assistance today?",
+    "I'm okay and yourself": "Glad to know! How can i be of assistance today?",
+    "Where are you currently located": "I am currently located in Johannesburg, Gauteng",
+    "What services do you offer": "I offer web development, web design,mobile dev , mobile design, and SEO services.",
+    "Can you tell me about yourself": "I'm a software developer with over 2 years in web dev and over 3 years in mobile app dev currently based in Johannesburg Gauteng",
+    "How can I contact you": "You can reach me through the contact form on this website or my personal email address: borekamohelo@gmail.com",
+    "What projects have you worked on": "I have worked on various web development projects, Mobile Appliacation Projects also including design. For more visit: https://github.com/NuffSaid-Bore?tab=repositories",
+    "What is your experience": "I have several years of experience in web development, mobile development and recenty started with design.",
+    "Goodbye": "Goodbye! Have a great day!",
+};
+
+const optionResponses = [
+    { text: "What services do you offer?", value: "What services do you offer?" },
+    { text: "Can you tell me about yourself?", value: "Can you tell me about yourself?" },
+    { text: "How can I contact you?", value: "How can I contact you?" },
+    { text: "Where are you currently located?", value: "How can I contact you?" },
+    { text: "Goodbye", value: "Goodbye" }
+];
+
+const createChatLi = (message, className) => {
     const chatli = document.createElement("li");
     chatli.classList.add("chat", className);
-
-    let chatContent = className === "outgoing" ? `<p>${message}</p>` : ` <i class='bx bxs-bot'></i><p>${message}</p>`;
+    let chatContent = className === "outgoing" 
+        ? `<p>${message}</p>` 
+        : `<i class='bx bxs-bot'></i><p>${message}</p>`;
     chatli.innerHTML = chatContent;
     return chatli;
 }
```

Comment:
you can add a hyperlink to your responses so that user's can navigate to relevant pages on your website

Reply:
Thanks for the suggestion. I am working on it